src/applications/discover-your-benefits/tests/unit/container/ConfirmationPage.unit.spec.jsx

```diff
@@ -165,6 +166,29 @@ describe('ConfirmationPage - sortBenefits and filterBenefits', () => {
     expect(benefitNames[1]).to.contain('Education');
   });
 
+  it('should sort benefits alphabetically by default', () => {
+    wrapper = setup({
+      results: { data: BENEFITS_LIST },
+      location: {
+        basename: '/',
+        pathname: '/confirmation',
+        query: { allBenefits: true },
+      },
+    });
+    container = wrapper.container;
+
+    const benefitNames = wrapper
+      .getAllByRole('listitem')
+      .map(li => li.textContent);
+
+    const sortedBenefits = BENEFITS_LIST.sort((a, b) => {
+      return a.name.localeCompare(b.name);
+    });
+    for (let i = 0; i < benefitNames.length; i++) {
+      expect(benefitNames[i].name === sortedBenefits.name).to.be.true;
+    }

```

Comment:
```suggestion
    benefitNames.forEach(benefit => {
      expect(benefit.name).to.equal(sortedBenefits.name);
    });

```

Minor suggestion for using the ES6 array function and some refactoring to make it a little cleaner. 

Small nit, it might make more sense for `benefitNames` to be named `benefits` or something more abstract since that array doesn't contain just the benefit name but other info about the benefits also.